views/dashboard.handlebars

```diff
@@ -0,0 +1,34 @@
+<link rel="stylesheet" href="/static/css/dashboard.css">
+<script src="/static/js/dashboard.js"></script>
+
+<div id="dashboard-title">
+  {{ id }}'s Dashboard
+</div>
+
+<dashboard-section>
+  <dashboard-nav>
+    <ul>
+      <li><a href="/dashboard/{{ id }}">Dashboard</a></li>
+      <li><a href="#">Edit Categories</a></li>
```

Comment:
Don't include "Edit Categories" in the list for this PR until we have a view created for it to link to. You can add it back to the list once we have that view created. We want each change to have "functionality" and right now this link doesn't have functionality because it directs user to the same page and not the page we intend for "Edit Categories".